locale/en_US/admin.po

```diff
@@ -536,36 +563,84 @@ msgstr "Did you mean one of the following?"
 msgid "admin.jobs.totalCount"
 msgstr "There's a total of <strong>{$total}</strong> job(s) on the queue"
 
+msgid "admin.jobs.failed.totalCount"
+msgstr "There's a total of <strong>{$total}</strong> failed job(s)."
+
 msgid "admin.jobs.viewQueuedJobs"
 msgstr "View queued jobs"
 
 msgid "navigation.tools.jobs"
 msgstr "Jobs"
 
+msgid "navigation.tools.jobs.failed"
+msgstr "Failed Jobs"
+
+msgid "navigation.tools.jobs.failed.details"
+msgstr "Failed Job Details"
+
 msgid "navigation.tools.jobs.description"
 msgstr "View all of the queued jobs in the system and track failed attempts."
 
 msgid "navigation.tools.jobs.view"
 msgstr "View Jobs"
 
+msgid "navigation.tools.jobs.failed.view"
+msgstr "View Failed Jobs"
+
+msgid "navigation.tools.job.failed.details.view"
+msgstr "View Failed Job:{$id} Details"
+
+msgid "admin.job.failed.list.attribute"
+msgstr "Attribute"
+
+msgid "admin.job.failed.list.attribute.value"
+msgstr "Attribute Value"
+
 msgid "admin.jobs.list.id"
 msgstr "ID"
 
 msgid "admin.jobs.list.displayName"
 msgstr "Job"
 
-msgid "admin.jobs.list.queueName"
+msgid "admin.jobs.list.queue"
 msgstr "Queue"
 
+msgid "admin.jobs.list.connection"
+msgstr "Connection"
+
 msgid "admin.jobs.list.attempts"
 msgstr "Attempts"
 
 msgid "admin.jobs.list.createdAt"
 msgstr "Created At"
 
+msgid "admin.jobs.list.failedAt"
+msgstr "Failed At"
+
+msgid "admin.jobs.list.payload"
+msgstr "Payload"
+
+msgid "admin.jobs.list.exception"
+msgstr "Exception"
+
+msgid "admin.jobs.list.actions"
+msgstr "Actions"
+
+msgid "admin.jobs.failedAt"
+msgstr "Created at {$failedAt}"
+
 msgid "admin.jobs.createdAt"
 msgstr "Created at {$createdAt}"
 
+msgid "admin.jobs.failed.action.redispatch"
+msgstr "Redispatch"
+
+msgid "admin.jobs.failed.action.delete"
+msgstr "Delete"
```

Comment:
You can re-use an existing locale key for "delete".

Reply:
done .